Gears/script.cpp

```diff
@@ -110,6 +111,10 @@ void functionHShiftKeyboard();
 void functionHShiftWheel();
 void functionSShift();
 void functionAShift();
+float getGearMinSpeed(int gear);
```

Comment:
You might want to refactor your TCU-specific things into a new file (and namespace, if you fancy). I plan to move the transmission stuff out of script.cpp *some* day, like I did with other stuff.